app/master/build_request_handler.py

```diff
@@ -148,7 +149,13 @@ def _compute_subjobs_for_build(self, build_id, job_config, project_type):
         :type project_type: project_type.project_type.ProjectType
         :rtype: list[Subjob]
         """
-        atoms_list = job_config.atomizer.atomize_in_project(project_type)
+        # Users can override the list of atoms to be run in this build. If the atoms_override
+        # was specified, we can skip the atomization step and use those overridden atoms instead.
+        if project_type.atoms_override is not None:
+            atoms_string_list = project_type.atoms_override
+            atoms_list = [Atom(atom_string_value) for atom_string_value in atoms_string_list]
```

Comment:
Do you think we should do any prevalidation that the overridden atoms are valid or should we just let the build go ahead and fail slave-side? I suppose it would be the same effect as a misconfigured job config in that case (e.g., specifying different atomizer variable in the commands section and the atomizer section).